packages/devtools_app/lib/src/timeline/flutter/timeline_flame_chart.dart

```diff
@@ -264,25 +264,30 @@ class _FullTimelineFlameChartState
       if (row >= rows.length) {
```

Comment:
rename this helper and use it everywhere we are adding new rows at the bottom to fit a new required number of rows.
As a nit, consider just adding the rows directly instead of using addAll and List.generate.
That reduces the # of allocations and is maybe a bit clearer. For example
```dart
for(int i = rows.length; i < newRowLength; ++i) {
  rows.add(FlameCharRow(nodes: []));
}
```